### ISSUE TITLE
trtexec   --exportProfile  profile files only have one layer of information, and the model structure cannot be restored.

### ISSUE THREAD

**Issue by 18liumin** [NVIDIA/TensorRT#3768]
![image](https://github.com/NVIDIA/TensorRT/assets/51792451/8bea3ffa-360c-4130-ad64-9a845c7eeaf5)

If there is torch.cat, there will be a problem that the model structure has only one layer.

**Comment by zerollzeng** [comment]
Because they are fused by TRT so you can only see a layer with some name like myelin_xxx?

**Comment by 18liumin** [comment]
engine.profile.json  has {ForeignNode[encoder.fea_bn1.weight.../softmax/Softmax]}

**Comment by 18liumin** [comment]
> Because they are fused by TRT so you can only see a layer with some name like myelin_xxx?

engine.profile.json has {ForeignNode[encoder.fea_bn1.weight.../softmax/Softmax]}

**Comment by zerollzeng** [comment]
Yes this is the fused optimized graph.

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks per our policy, thanks all!
